presto-orc/src/main/java/com/facebook/presto/orc/metadata/statistics/BloomFilter.java

```diff
@@ -224,52 +222,7 @@ public static void serialize(OutputStream out, BloomFilter bloomFilter) throws I
         }
     }
```

Comment:
Refactoring this class was on my to-do list, we can also refactor and remove the method "serialize".